src/storage/btree.c

```diff
@@ -6222,9 +6228,16 @@ btree_get_unique_statistics (THREAD_ENTRY * thread_p, BTID * btid, int *oid_cnt,
 
   assert ((root_header->unique_pk & (BTREE_CONSTRAINT_UNIQUE | BTREE_CONSTRAINT_PRIMARY_KEY)) != 0);
 
-  *oid_cnt = root_header->num_oids;
-  *null_cnt = root_header->num_nulls;
-  *key_cnt = root_header->num_keys;
+  *oid_cnt = (unsigned int) root_header->num_oids;
+  *null_cnt = (unsigned int) root_header->num_nulls;
+  *key_cnt = (unsigned int) root_header->num_keys;
+
+  if (root_header->_64.over)
+    {
+      *oid_cnt |= (((long long) root_header->_64.num_oids) << 32);
```

Comment:
Bit operation was performed after the 10-bit memory was converted to 64-bit. This code looks a bit dangerous, have any tests been performed to verify this part?

Reply:
Yes, the code is verfied enough.